store/src/main/java/com/nytimes/android/external/store/base/impl/RealInternalStore.java

```diff
@@ -227,6 +231,11 @@ void notifySubscribers(Parsed data) {
         return stream;
     }
 
+    @NonNull @Override
+    public Observable<Parsed> stream() {
+        return subject.asObservable();
```

Comment:
lets add a `.share` before giving this out, should have been there all along 
http://blog.danlew.net/2016/06/13/multicasting-in-rxjava/